### ISSUE TITLE
[Trader] Pause and Resume

### ISSUE THREAD

**Issue by Herklos** [Drakkar-Software/OctoBot-Trading#55]
Proposal : 
- Create an annotation for-each concerned methods that pass if Trader is paused.

**Comment by GuillaumeDSM** [comment]
Yep, good idea, I'm seeing consumer callbacks that are in this case, are there many other methods ?

**Comment by Herklos** [comment]
No I don't think so, maybe 4-5 methods.
